.rhdh/scripts/install-rhdh-latest-build.sh

```diff
@@ -0,0 +1,109 @@
+# code yoinked from https://gitlab.cee.redhat.com/rhidp/rhdh/-/blob/rhdh-1-rhel-9/build/scripts/installCatalogSourceFromIIB.sh
```

Comment:
rather than two nearly identical code snippets why don't we move https://gitlab.cee.redhat.com/rhidp/rhdh/-/blob/rhdh-1-rhel-9/build/scripts/installCatalogSourceFromIIB.sh into this repo ?

Reply:
That makes sense - will do that later today. Thanks!

Reply:
Fixed in [`953a590` (#184)](https://github.com/janus-idp/operator/pull/184/commits/953a5909573a2088d125c0cf7dd2f77e41b28752). This commits adds an `install-rhdh-catalog-source.sh` script, largely inspired from https://gitlab.cee.redhat.com/rhidp/rhdh/-/blob/rhdh-1-rhel-9/build/scripts/installCatalogSourceFromIIB.sh, but trying to simplify what I could.
PTAL.
Moving to ImageDigestMirrorSet can be done in a separate PR.